src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -26,126 +27,392 @@ function oneYearFromNow() {
-                  id: false,
-                  timestamp: false,
-                  hasAttemptedSubmit: false,
+    it('sorts the in-progress applications, listing the soonest-to-expire applications first', () => {
+      const initialState = {
+        user: {
+          profile: {
+            savedForms: [
+              {
+                form: '686C-674',
+                metadata: {
+                  version: 1,
+                  returnUrl: '/net-worth',
+                  savedAt: oneDayAgo(),
+                  submission: {
+                    status: false,
+                    errorMessage: false,
+                    id: false,
+                    timestamp: false,
+                    hasAttemptedSubmit: false,
+                  },
+                  expiresAt: oneWeekFromNow() / 1000,
+                  lastUpdated: oneDayAgo() / 1000,
+                  inProgressFormId: 5179,
                 },
-                expiresAt: oneWeekFromNow() / 1000,
                 lastUpdated: oneDayAgo() / 1000,
-                inProgressFormId: 5179,
               },
-              lastUpdated: oneDayAgo() / 1000,
-            },
-            {
-              form: '21-526EZ',
-              metadata: {
-                version: 1,
-                returnUrl: '/net-worth',
-                savedAt: oneDayAgo(),
-                submission: {
-                  status: false,
-                  errorMessage: false,
-                  id: false,
-                  timestamp: false,
-                  hasAttemptedSubmit: false,
+              {
+                form: '21-526EZ',
+                metadata: {
+                  version: 1,
+                  returnUrl: '/net-worth',
+                  savedAt: oneDayAgo(),
+                  submission: {
+                    status: false,
+                    errorMessage: false,
+                    id: false,
+                    timestamp: false,
+                    hasAttemptedSubmit: false,
+                  },
+                  expiresAt: oneDayFromNow() / 1000,
+                  lastUpdated: oneDayAgo() / 1000,
+                  inProgressFormId: 5179,
                 },
-                expiresAt: oneDayFromNow() / 1000,
                 lastUpdated: oneDayAgo() / 1000,
-                inProgressFormId: 5179,
               },
-              lastUpdated: oneDayAgo() / 1000,
-            },
-            {
-              form: '1010ez',
-              metadata: {
-                version: 1,
-                returnUrl: '/net-worth',
-                savedAt: oneDayAgo(),
-                submission: {
-                  status: false,
-                  errorMessage: false,
-                  id: false,
-                  timestamp: false,
-                  hasAttemptedSubmit: false,
+              {
+                form: '1010ez',
+                metadata: {
+                  version: 1,
+                  returnUrl: '/net-worth',
+                  savedAt: oneDayAgo(),
```

Comment:
Can you explain why we divide some timestamps by `1000`, and not others?

Reply:
to mirror what we get from the API. savedAt is in ms, the others are all in seconds. 
<img width="392" alt="Screen Shot 2021-03-04 at 9 07 38 AM" src="https://user-images.githubusercontent.com/20728956/110001446-3dadc080-7cc9-11eb-90ca-b320a94735a3.png">

Reply:
Ah, thank you! Weird how they are different.